dashboard/dashboard-intro.md

```diff
@@ -5,6 +5,10 @@ category: how-to
 
 # TiDB Dashboard 介绍
 
+> **注意：**
+>
+> 该功能目前为实验特性，不建议在生产环境中使用。
```

Comment:
However actually we **recommends** to use in production 🤔

Reply:
The feedback from the QA team is that it is better to wait for them to test it and then remove this warning.

Reply:
So is it okay to merge this for now? @breeswish